apps/transport/priv/gettext/page-dataset-details.pot

```diff
@@ -393,3 +393,7 @@ msgstr ""
 #, elixir-format
 msgid "time to live (%{feed})"
 msgstr ""
+
+#, elixir-format
+msgid "download availability over last %{nb} days"
```

Comment:
je pense qu'on dit plutôt "over **the** last xxx days"

Reply:
https://ell.stackexchange.com/questions/98377/over-the-past-two-days-or-in

Reply:
Merci de ta vigilance ! Corrigé dans https://github.com/etalab/transport-site/pull/2032/commits/6a8232b3f9bf022e7d642e2e5089422e475e9abb